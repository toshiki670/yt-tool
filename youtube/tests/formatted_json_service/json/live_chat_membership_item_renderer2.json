{
  "clickTrackingParams": "CAEQl98BIhMIz5X5ur27igMVoc80Bx03uTj5",
  "replayChatItemAction": {
    "actions": [
      {
        "clickTrackingParams": "CAEQl98BIhMIz5X5ur27igMVoc80Bx03uTj5",
        "addChatItemAction": {
          "item": {
            "liveChatMembershipItemRenderer": {
              "id": "ChwKGkNMYTh5dktvdTRvREZYckV3Z1FkalRvN01B",
              "timestampUsec": "1734868527656597",
              "timestampText": { "simpleText": "-4:30" },
              "authorExternalChannelId": "UCVizv_lHkXIpFD9jkAeOMbw",
              "headerSubtext": {
                "runs": [
                  { "text": "Welcome to " },
                  { "text": "へい" },
                  { "text": "!" }
                ]
              },
              "authorName": { "simpleText": "り" },
              "authorPhoto": {
                "thumbnails": [
                  {
                    "url": "https://yt4.ggpht.com/NqYjLO8GjaIuNEJ6TUgfOwWNO9gb6P-ACRKOO9nPmlU94jyP1iet8CsqxM8Mu6u6X-8iEbjw=s32-c-k-c0x00ffffff-no-rj",
                    "width": 32,
                    "height": 32
                  },
                  {
                    "url": "https://yt4.ggpht.com/NqYjLO8GjaIuNEJ6TUgfOwWNO9gb6P-ACRKOO9nPmlU94jyP1iet8CsqxM8Mu6u6X-8iEbjw=s64-c-k-c0x00ffffff-no-rj",
                    "width": 64,
                    "height": 64
                  }
                ]
              },
              "authorBadges": [
                {
                  "liveChatAuthorBadgeRenderer": {
                    "customThumbnail": {
                      "thumbnails": [
                        {
                          "url": "https://yt3.ggpht.com/c9FVnU3zQyRhy7W4z_vm4hGF7bjSyXcyxIVTELl9dDNndD-tXfDNCjRQxHfgZu-NCHq1j4VmEQ=s16-c-k",
                          "width": 16,
                          "height": 16
                        },
                        {
                          "url": "https://yt3.ggpht.com/c9FVnU3zQyRhy7W4z_vm4hGF7bjSyXcyxIVTELl9dDNndD-tXfDNCjRQxHfgZu-NCHq1j4VmEQ=s32-c-k",
                          "width": 32,
                          "height": 32
                        }
                      ]
                    },
                    "tooltip": "Member (1 year)",
                    "accessibility": {
                      "accessibilityData": { "label": "Member (1 year)" }
                    }
                  }
                }
              ],
              "contextMenuEndpoint": {
                "clickTrackingParams": "CJkCEOD9BiITCM-V-bq9u4oDFaHPNAcdN7k4-Q==",
                "commandMetadata": {
                  "webCommandMetadata": { "ignoreNavigation": true }
                },
                "liveChatItemContextMenuEndpoint": {
                  "params": "Q2g0S0hBb2FRMHhoT0hsMlMyOTFORzlFUmxoeVJYZG5VV1JxVkc4M1RVRWFLU29uQ2hoVlExcHNSRmg2UjI5dk4yUTBOR0ozWkU1UFlrWmhZMmNTQzBsb1NsY3RXVkJLTFhvd0lBRW9BVElhQ2hoVlExWnBlblpmYkVocldFbHdSa1E1YW10QlpVOU5ZbmM0QWtnQlVBUSUzRA=="
                }
              },
              "contextMenuAccessibility": {
                "accessibilityData": { "label": "Chat actions" }
              },
              "trackingParams": "CJkCEOD9BiITCM-V-bq9u4oDFaHPNAcdN7k4-Q=="
            }
          },
          "clientId": "CLa8yvKou4oDFXrEwgQdjTo7MA"
        }
      }
    ],
    "videoOffsetTimeMsec": "0"
  }
}
