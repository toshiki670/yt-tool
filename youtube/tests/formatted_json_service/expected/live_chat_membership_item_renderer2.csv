id,authorExternalChannelId,postedAt,authorName,content,isModerator,membershipMonths,category
ChwKGkNMYTh5dktvdTRvREZYckV3Z1FkalRvN01B,UCVizv_lHkXIpFD9jkAeOMbw,2024-12-22T20:55:27.656597+09:00,り,Welcome to へい!,false,Member (1 year),ChatMembershipItem
